docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -149,38 +160,35 @@ If you observe that the pipeline is not making expected progress, check the Data
 
 In this case, check if you observe the following:
 
-- ***The watermark of the Spanner to Sink pipeline does not advance***
-
-    This happens when the job is hit with a huge backlog, that leads to infinite loop. The recovery steps are covered [here](#recovery-steps-for-the-infinte-loop).
 
-- ***PubSub message count is not decreasing and the same data is being written back to source repeatedly***
-
-    This happens when the time to write all messages to source database and send an ACK to PubSub exceeds the deadline. Ensure the ACK deadline for the subscriptions are high enough (10 minutes is the highest value, consider bumping it to that). If still facing this issue, consider moving the Dataflow job writing to source database geographically closer to the source database.
-
-- ***There is data in change stream yet not present in Pub/Sub***
+- ***There is data in change stream yet not present in GCS***
 
     Records of below nature are dropped from reverse replication. Check the Dataflow logs to see if they are dropped.
```

Comment:
We dont talk about the skip directory here?

Reply:
Ah, good point. Thanks.Added.

Reply:
Ah, good point. Thanks.Added.